internal/app/app.go

```diff
@@ -215,18 +215,16 @@ func runDsync(c *cli.Context) error {
 	}
 
 	if needWebServer {
-		//start a web server to serve progress report
+		// Start a web server to serve progress report
 		go func() {
 			host := fmt.Sprintf("localhost:%d", o.WebPort)
-			slog.Info("Starting web server to serve progress report on " + host + "/progress")
-			fs := http.FileServer(http.Dir("./web_static"))
-
+			slog.Info("Starting web server to serve progress report on " + host)
+			// Generate initial static page
+			http.Handle("/", http.FileServer(http.Dir("./web_static")))
```

Comment:
hmm we can probably change how we do this so we embed the files in the go binary, that way we don't have to think about this directory when packaging the software.

https://pkg.go.dev/embed